VALUES.md

```diff
@@ -0,0 +1,25 @@
+# Core Values
+
+## Our Mission
+
+At its core, our project is a collection of people working towards a shared goal.
```

Comment:
This line should end with a colon rather than a period, I think. Otherwise, the next sentence is a fragment.